SQL/New_patches/2020-04-01-GUID.sql

```diff
@@ -0,0 +1,16 @@
+ALTER Table Project ADD COLUMN GUIDRequired binary NOT NULL DEFAULT 0;
+ALTER Table candidate ADD COLUMN GUID VARCHAR(100) DEFAULT NULL;
+
+INSERT INTO ConfigSettings (Name, Description, Visible, AllowMultiple, DataType, Parent, Label, OrderNumber)
+SELECT 'piiURL', 'Address of server hosting the personally identifying information', 1, 0, 'text', MAX(Parent), 'PII URL', MAX(OrderNumber)+1
```

Comment:
I don't think any server is supposed to have any kind of PII unless the design has changed radically since I was last involved. The point of this whole thing is that PII never gets stored (or even transferred over HTTP) to preserve privacy.

For this reason I'd avoid using PII in any comments or variable names because they're always going to seem dangerous.